### PR TITLE
Add kubernetes NPD adaption to ABRT post

### PR DESCRIPTION
**Pull request by juliusmilan**
Post with kubernetes node-problem-detector ABRT-adaption config, with link to short demo on youtube.

### CODE REVIEW

File: _posts/2017-03-31-kubernetes-npd-adaptation.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+author: Julius Milan <jmilan@redhat.com>
+visible: 1
+categories: abrt kubernetes node-problem-detector
+---
+Monitoring of failures in kubernetes nodes is now improved once more.
+
+Part of the kubernetes project, the [node-problem-detector][npd-link]
+which purpose is to report various problems occuring in kubernetes nodes to the
+upstream layers in cluster management stack is now able to report problems detected
+by ABRT. Basicly any kind of problem detected by ABRT (Uncaught Python / Java or
```
Comment (xsuchy) [inline _posts/2017-03-31-kubernetes-npd-adaptation.markdown:14]: Basicaly

Comment (marusak) [reply _posts/2017-03-31-kubernetes-npd-adaptation.markdown:14]: Basically :)

File: _posts/2017-03-31-kubernetes-npd-adaptation.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+layout: post
+title:  "ABRT catched problems reporting to kubernetes"
+date:   2017-03-31 14:03:18 +0100
+author: Julius Milan <jmilan@redhat.com>
+visible: 1
+categories: abrt kubernetes node-problem-detector
+---
+Monitoring of failures in kubernetes nodes is now improved once more.
+
+Part of the kubernetes project, the [node-problem-detector][npd-link]
```
Comment (xsuchy) [inline _posts/2017-03-31-kubernetes-npd-adaptation.markdown:11]: Kubernetes (capital K) - can be also a link.

**Comment by juliusmilan** [comment]
Closing this, new PR was created (https://github.com/abrt/abrt.github.io/pull/4) with the same change & updated, because of bad branch name (starting with -)

**Pull request closed without merging**